images/payload/hack/prepare.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
```

Comment:
Where is `hack/prepare` used?

Reply:
I'm not sure. @fidencio ? Is prepare.sh called manually? Or is it somehow used by Travis?